pvmismatch/pvmismatch_lib/pvmodule.py

```diff
@@ -167,12 +170,17 @@ def parse_diode_config(Vbypass, cell_pos):
         num_bypass = len(Vbypass)
     except TypeError:
         # float passed - default case - Vbypass across every cell string
-        return 'default'
+        return DEFAULT_BYPASS
     else:
         # if only one value is passed in the list- assume only one
         # bypass diode  across the PV module
         if len(Vbypass) == 1:
-            return 'module_bypass'
+            return MODULE_BYPASS
+        # if more than 1 values are passed, apply them across
+        # the cell strings in ascending order
+        elif len(cell_pos) == num_bypass:
+            return CUSTOM_SUBSTR_BYPASS
+
         # if more than 1 values are passed, apply them across
         # the cell strings in ascending order
         elif len(cell_pos) == num_bypass:
```

Comment:
Seems like this section will never execute, since elif statement is the same as the prior elif -- should the second section be removed?